packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -456,25 +449,18 @@ export function createEditPageNode({
                                     )}
                                 </Field>
                                 {!disableHideInMenu && (
-                                    <Field
-                                        label={intl.formatMessage({
+                                    <CheckboxField
+                                        fieldLabel={intl.formatMessage({
```

Comment:
@jamesricky is this the recommended way for the two labels?

Reply:
Yes, that's how it's done in the "All Field Components" story:

https://github.com/vivid-planet/comet/blob/22d4fdf6553c654b335720442e9d4cf0b17dac0f/storybook/src/admin/form/AllFieldComponents.tsx#L103-L114

<img width="635" alt="Screenshot 2024-09-26 at 12 45 24" src="https://github.com/user-attachments/assets/46faea29-c13e-44cb-808a-e39f98e9c05a">